lua/wire/client/text_editor/texteditor.lua

```diff
@@ -1505,6 +1504,7 @@ function EDITOR:CreateFindWindow()
 	pnl.GoToLineTab.Tab.OnMousePressed = function( ... )
 		pnl:SetHeight( 86 )
 		pnl.TabHolder:StretchToParent( 1, 23, 1, 1 )
+		pnl.GoToLineTab.Entry:SetText(self.Caret[1])
```

Comment:
Setting the text here is required if the dialog was opened on a different tab, like Find or Replace, so we set the text if the user switched to "Go to line" tab.